src/parser/jobs/nin/modules/Ninki.js

```diff
@@ -104,4 +120,26 @@ export default class Ninki extends Module {
 			</Trans>,
 		}))
 	}
+
+	output() {
+		const ninkiColor = Color(JOBS.NINJA.colour)
+
+		/* eslint-disable no-magic-numbers */
+		const chartdata = {
+			datasets: [
+				{
+					label: 'Ninki',
+					steppedLine: true,
+					data: this._ninkiHistory,
+					backgroundColor: ninkiColor.fade(0.8),
+					borderColor: ninkiColor.fade(0.5),
+				},
+			],
+		}
+		/* eslint-enable no-magic-numbers */
+
+		return <Fragment>
```

Comment:
for future ref, you can write fragments as `<> ... </>` instead of `<Fragment> ... </Fragment>`